CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Change Log
+
+This file documents all notable changes in `LogDNA CloudWatch Lambda Function`. The release numbering uses [semantic versioning](http://semver.org).
+
+## v2.0.0 - Released on June 5, 2019
+* Releasing new `Lambda Function` written/implemented on `node.js`
```

Comment:
This read weird to me - I'm more used to changelogs having past tense. Like "Released new...", "Added retry"... etc. E.g. https://github.com/lodash/lodash/wiki/Changelog

Reply:
Haha - done!